src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -245,6 +305,10 @@ private String getCheckShortName(TreeWalkerAuditEvent event) {
         return checkNames[checkNames.length - 1];
     }
 
+    private String getShortName(TreeWalkerAuditEvent event) {
```

Comment:
Confusing name, because we already have getCheckShortName().
Please rename:
`getCheckShortName()` -> `getCheckName()`
`getShortName()` -> `getCheckShortName()`

Reply:
done